platform/ui/src/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -88,6 +90,8 @@ ToolbarButton.propTypes = {
   /* Influences background/hover styling */
   type: PropTypes.oneOf(['action', 'toggle', 'tool']),
   id: PropTypes.string.isRequired,
+  defaultTool: PropTypes.string,
+  commands: PropTypes.object.isRequired,
```

Comment:
I am seeing a bunch of warnings/errors in the console as a result of adding `commands` here, please fix.